ClusterTest/src/main/scala/net/snowflake/spark/snowflake/TestUtils.scala

```diff
@@ -225,7 +225,7 @@ object TestUtils {
     jdbcProperties.put("client_session_keep_alive", "true")
 
     // Force DECIMAL for NUMBER (SNOW-33227)
-    jdbcProperties.put("JDBC_TREAT_DECIMAL_AS_INT", "false")
```

Comment:
https://github.com/ActionIQ-OSS/spark-snowflake/blob/5cd5e9ba6ee7ff332f436f440408b908608d3e86/ClusterTest/src/main/scala/net/snowflake/spark/snowflake/testsuite/ClusterTestSuiteBase.scala#L117

Some tests there doing snowflake reads, possible to add a decimal column there and then read it as int?